yt/odin/checks/bin/operations_snapshots/__main__.py

```diff
@@ -23,22 +14,37 @@ def run_check(yt_client, logger, options, states):
     batch_client = yt.create_batch_client(client=yt_client, max_batch_size=100)
 
     for operation in operations:
-        # TODO(asaitgalin): Migrate to new operations storage scheme.
-        op_path = "//sys/operations/" + operation
-        responses[(operation, "events")] = batch_client.get(op_path + "/@events")
-        responses[(operation, "last_successful_snapshot_time")] = \
-            batch_client.get(op_path + "/@progress/last_successful_snapshot_time")
+        args = {"attributes": ["events", "progress"]}
+        if operation.startswith("*"):
+            args["operation_alias"] = operation
+            args["include_scheduler"] = True
+        else:
+            args["operation_id"] = operation
+
+        responses[operation] = batch_client.get_operation(**args)
 
     batch_client.commit_batch()
 
     operations_without_snapshots = []
 
     for operation in operations:
-        events = process_response(responses[(operation, "events")])
-        last_successful_snapshot_time = process_response(
-            responses[(operation, "last_successful_snapshot_time")])
-
-        if events is None or last_successful_snapshot_time is None:
+        op_resp = responses[operation]
+
+        if not op_resp.is_ok():
+            error = yt.YtResponseError(op_resp.get_error())
+            if not error.is_resolve_error():
+                raise error
+            else:
+                logger.warning("Got error resolving %s: %s", operation, error)
```

Comment:
`Error resolving path for operation (operation_id: %s, error: %s)`

And I am not sure that it is necessary to log since it is normal situation that operation finished between list request and later get request